src/boot/O2/gfxprint.c

```diff
@@ -7,11 +7,7 @@
 #define GFXP_FLAG_ENLARGE (1 << 6)
 #define GFXP_FLAG_OPEN (1 << 7)
 
-//! TODO: Need to extract
-extern u16 sGfxPrintFontTLUT[64];
-extern u16 sGfxPrintRainbowTLUT[16];
-extern u8 sGfxPrintRainbowData[8];
-extern u8 sGfxPrintFontData[2048];
+#include "assets/boot/gfxprint/gfxprint.c"
```

Comment:
I would kinda prefer if we defined the C symbols on this file (using the corresponding includes) instead of including this generated file